exercises/isbn-verifier/isbn-verifier.spec.js

```diff
@@ -0,0 +1,81 @@
+import ISBN from './isbn-verifier.js';
+
+describe('ISBN Verifier Test Suite', () => {
+  test('valid isbn number', () => {
+    const isbn = new ISBN('3-598-21508-8');
+
+    expect(isbn.isValid()).toBeTruthy();
```

Comment:
Why '.toBeTruthy()' instead of `.toEqual(true)`?  Did you want to leave the tests open to someone returning something other than a boolean `true`?  Just curious...

Reply:
You're correct. In this case we only want to get true or false, so I've changed it to all use `.toEqual`